app/src/App/hooks.ts

```diff
@@ -120,6 +123,46 @@ export function useProtocolReceiptToast(): void {
   }, [protocolIds])
 }
 
+export function useModuleAttachedToast(launchModuleSetupCallback: () => void) {
+  const { t, i18n } = useTranslation(['module_wizard_flows', 'shared'])
+  const { makeToast } = useToaster()
+  const attachedModules = useAttachedModules()
+  const attachedPipettes = useAttachedPipettes()
+  const deckConfig = useNotifyDeckConfigurationQuery({
+    enabled: attachedModules.length > 0,
+  }).data
+  const moduleSerials = attachedModules
+    .filter(m => m.moduleOffset === undefined)
+    .map(m => m.serialNumber)
+  const moduleSerialsRef = useRef(moduleSerials)
+
+  useEffect(() => {
+    if (deckConfig === undefined) return
```

Comment:
```suggestion
    if (deckConfig != null) {
    ....
  }
```
Nit:

1. We prefer loose equality checks with `null` for the absence of something. This is just a code preference thing, but it does have practical implications if we ever refactor the deck config hook to return `null`: we wouldn't have to refactor these strict equality checks.
2. If you want to do 
```
if (deckConfig == null) {
return}
```
as an alternative, I'd get in the habit of including braces, even for one-line conditional checks. It's reduces the likelihood of the occasional bug where braces are expected and they are dropped if we just use braces everywhere.